packages/news-api/src/index.ts

```diff
@@ -48,7 +43,7 @@ const fetcher = async (
     callback: (statusCode: number, data: string | null, errMessage?: string) => void,
 ) => {
     axios
```

Comment:
unrelated. just wondering we use axios here. it is imho completely unnecessary.

Reply:
@vladimirvolek Do you remember why you've chosen axios here? Is it a good idea to drop axios and use whatwg-fetch?

Reply:
You can drop it if you want and use something else.

Reply:
Btw are you sure the environments are not really used?

https://staging-news.trezor.io/
https://news.trezor.io/

Reply:
https://github.com/trezor/trezor-suite/blob/b010f7a5556be360a6bf9b4a34efebba570cd017/packages/suite/src/hooks/dashboard/useNews.ts#L16

Reply:
Thanks, addressed in 725965616cf469d41322b1c0460b4eb09cb26a3b